fabric_plugin/tests/tests.py

```diff
@@ -412,6 +428,11 @@ def setUp(self):
             if files.exists(self.CUSTOM_BASE_DIR):
                 api.run('rm -rf {0}'.format(self.CUSTOM_BASE_DIR))
 
+    def test_run_script_as_sudo(self):
+        self._execute(
+            'test_sudo.run_script',
+            script_path='scripts/mkdir.sh')
```

Comment:
can you add an assert to the end of the test that the directory the script created, indeed exists?